jobbergate/cli.py

```diff
@@ -368,7 +368,7 @@ def _wrapper(**kvargs):
             file = outputfile.write(jinjatemplate.render(data=data))
             outputfile.flush()
             if "cmd_command" in data.keys():
-                subprocess.run(data["cmd_command"].split(), stdout=True, check=True)
+                os.system(data["cmd_command"])
             return file
```

Comment:
I try to read up on what's best. `subprocess.call(data["cmd_command"], shell=True)` or `subprocess.Popen(data["cmd_command"], shell=True)` may be the best solution since it takes the full string but still gives you the benefits that subprocess gives you with more controll and exceptions if it fails. I'm not against the os.system-call, and it could be that this extra control is not needed and os.system is the best and simplest.
Anyway, if you think os.system is the way to go I'll merge this.

Reply:
I changed to os.system to use its built in parsing. As I understand, it treats the input string as the os would, but subprocess needs the input to be cut into chewable pieces. 
We could wrap it in a try-except if you want, to increase the control we have over the output.

Reply:
Doesn't `shell=True` mimic the behaviour of os.system with regards to how the command and argument should be passed?

Reply:
You're right, I didn't know the _shell_ option allowed another type of input. I'll switch it back to subprocess, but with _shell=True_.